prep-javascript-variables/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    /* your code here */
+    var fullName = "Gaven Loughridge";
+    console.log('Full name: '+ fullName);
+    var numberOfPets = 1;
+    console.log('Pets: ' + numberOfPets);
+    var likesVegetables = false;
+    console.log('Likes Veggies: ' + likesVegetables);
+    var nothing = null;
+    var never;
+    console.log('Nothing: ' + nothing);
+    console.log('Never: ' + never);
```

Comment:
Try passing two separate arguments to `console.log()` so that the variables aren't coerced into strings. It can get dicey with objects later on `[object Object]` 👀